ts/src/profiler.ts

```diff
@@ -297,6 +298,7 @@ export class Profiler extends common.ServiceObject {
    */
   async profileAndUpload(prof: RequestProfile): Promise<void> {
     prof = await this.profile(prof);
+    this.logger.debug(`Successfully collected profile ${prof.profileType}`);
```

Comment:
Nit: this and some other log messages do not have the full stop, while others have. Please be consistent. I think it is common to omit the stop when the whole message is a single sentence and use the full stops when it's several sentences.